fluentlenium-kotest/src/main/kotlin/org/fluentlenium/adapter/kotest/FluentDescribeSpec.kt

```diff
@@ -0,0 +1,45 @@
+package org.fluentlenium.adapter.kotest
+
+import io.kotest.core.spec.style.DescribeSpec
+import org.fluentlenium.adapter.IFluentAdapter
+import org.fluentlenium.adapter.TestRunnerAdapter
+import org.fluentlenium.adapter.exception.AnnotationNotFoundException
+import org.fluentlenium.adapter.kotest.internal.KoTestFluentAdapter
+import org.fluentlenium.configuration.Configuration
+import org.fluentlenium.configuration.ConfigurationFactoryProvider
+
+abstract class FluentDescribeSpec internal constructor(
+    private val fluentAdapter: KoTestFluentAdapter,
+    body: FluentDescribeSpec.() -> Unit = {}
+) : DescribeSpec({ }),
+    IFluentAdapter by fluentAdapter,
+    TestRunnerAdapter {
+
+    constructor(body: FluentDescribeSpec.() -> Unit = {}) : this(KoTestFluentAdapter(), body)
+
+    init {
+        fluentAdapter.useConfigurationOverride = { configuration }
+
+        listener(fluentAdapter.listener())
+
+        body()
+    }
+
+    private val config: Configuration by lazy {
+        ConfigurationFactoryProvider.newConfiguration(javaClass)
+    }
+
+    override fun getConfiguration(): Configuration = config
+
+    override fun getTestClass(): Class<*> = javaClass
```

Comment:
You might be able to push a bunch of these methods up into an AbstractKotestFluentSpec or something.